extensions/_oci.md

```diff
@@ -0,0 +1,79 @@
+# _oci Extension Endpoints
+
+## Summary
+
+This base extension namespace for OCI namespaced extension endpoints.
+
+## Reference Explanation
+
+### Component: `ext`
+
+This component is for endpoints relating to extensions on the registry API being queried.
+
+#### Module: `discover`
+
+This endpoint may be queried to discover extensions available on this registry API.
+
+Registry-level extensions may be discovered with a standard `GET` as follows.
+
+```HTTP
+GET /v2/_oci/ext/discover
+```
+
+Repository-level extensions may be discovered with a standard GET as follows.
+
+```HTTP
+    GET /v2/{name}/_oci/ext/discover
+```
+
+The base extension returns an array of supported extensions with details of the endpoints as shown below.
+
+```HTTP
+200 OK
+Content-Length: <length>
+Content-Type: application/json
+
+{
+    "extensions": [
+        {
+            "name": "_<extension>",
+            "description": "",
+            "url": "..."
+        }
+    ]    
+}
+```
+
+### *Extensions* Property Descriptions
+
+- **`extensions`** *array of extension objects*, REQUIRED
+
+  This property contains a list of supported extension endpoints.
+
+  - **`name`** *string*, REQUIRED
+
+    The name of the extension (i.e. '`_oci`') as it will appear in the URL path.
+
+  - **`url`** *string*, REQUIRED
+
+    URL link to the documentation defining this extension and its endpoints.
+
+  - **`description`** *string*, OPTIONAL
+
+    Human readable description of this endpoint.
+
+  - **`endpoints`** *array of strings*, REQUIRED
```

Comment:
Are "endpoints" a reference to "components"?

Reply:
ah ok. we need examples

Reply:
For anyone that wasn't on the call today, endpoints refers to `_<extension>/<component>/<module>`. Not sure if we want to include `/v2/` and `/v2/<repository>` in there.